cylc/flow/scripts/check_versions.py

```diff
@@ -20,17 +20,17 @@
 
 Check that Cylc versions match on different platforms.
 
-Check the version of cylc invoked on each of WORKFLOW's task host platforms
-when CYLC_VERSION is set to *the version running this command line tool*.
+Check the version of Cylc invoked on each of workflow's task host platforms
+when CYLC_VERSION is set to <bold>the version running this command line
+tool</bold>.
```

Comment:
I am a bit unclear on what it means by "check the version... when CYLC_VERSION is set to..."

Reply:
It is checking that the remote versions are set to the local version as defined via CYLC_VERSION

Reply:
Reworded.